src/Dialogs/AdjustDateTimeDialog.vala

```diff
@@ -200,31 +199,26 @@ public class AdjustDateTimeDialog : Granite.Dialog {
         if (no_original_time)
             return;
 
-        original_time_label.set_text (_ ("Original: ") +
-                                      Time.local (original_time).format (use_24_hr_format ? _ ("%m/%d/%Y, %H:%M:%S") :
-                                              _ ("%m/%d/%Y, %I:%M:%S %p")));
+        original_time_label.set_text (
+            _ ("Original: ") +
+            (new DateTime.from_unix_local (original_time)).format (
+                use_24_hr_format ? _ ("%m/%d/%Y, %H:%M:%S") : _ ("%m/%d/%Y, %I:%M:%S %p")
+            )
```

Comment:
Not really related with your PR but something like this is better for the i10n:
```suggestion
            _ ("Original: %s").printf (
                (new DateTime.from_unix_local (original_time)).format (
                    use_24_hr_format ? _ ("%m/%d/%Y, %H:%M:%S") : _ ("%m/%d/%Y, %I:%M:%S %p")
                )
            )
```

Reply:
Yeah, I was studiously ignoring the myriad other coding issues :wink: